src/browser/types.ts

```diff
@@ -33,6 +33,12 @@ declare global {
     namespace WebdriverIO {
         type BrowserCommand = Exclude<FunctionProperties<WebdriverIO.Browser>, EventEmitterMethod>;
         type ElementCommand = Exclude<FunctionProperties<WebdriverIO.Element>, EventEmitterMethod>;
+        // eslint-disable-next-line @typescript-eslint/no-explicit-any
+        type OverwriteCommandFn<ThisArg, Fn extends (...args: any) => any> = (
+            this: ThisArg,
+            origCommand: OmitThisParameter<Fn>,
```

Comment:
Removing `this` parameter from the function we are overwriting, so we can call it without `bind`/`call`/`appy` functions:

```ts
browser.overwriteCommand('url', function (origCommand, path) {
    return origCommand(path); // no `call`/`bind`/`apply` is needed
})
```